rtrlib/rtr/rtr.h

```diff
@@ -132,6 +133,8 @@ struct rtr_socket {
 	unsigned int version;
 	bool has_received_pdus;
 	struct spki_table *spki_table;
+	struct ordered_dyn_array *aspa_array;
+	struct aspa_table *aspa_table;
```

Comment:
Why do we need the reference back to the main structure holding the arrays ?

Reply:
could you please have a look at my comment regarding the data structure issue? I really think we manouvred ourselves into corner here

Reply:
I'd say: yes, we don't need it, as it's the same as the global one anyways. however, it doesn't really hurt either to keep it for consistency's sake.